web/src/main/webapp/resources/js/atsy-new-application-popup.js

```diff
@@ -0,0 +1,46 @@
+var positionDefaultValueInEng = 'Please choose a position';
+var applicationSourceDefaultValueInEng = 'Please choose an application source';
+var positionDefaultValueInHun = 'Kérem válasszon egy pozíciót';
+var applicationSourceDefaultValueInHun = 'Kérem válasszon egy jelentkezési forrást';
+
+function validateApplicationSourceAndPositionDropDowns() {
+    var positionIsValid = positionValidator();
+    var applicationSourceIsValid = applicationSourceValidator();
+    return (positionIsValid && applicationSourceIsValid);
+}
+
+function positionValidator() {
+    var ddl = document.getElementById('position');
```

Comment:
Let's start using jQuery's features. You can use `$("#position")` instead. Please do the refactoring for the whole file.